app/lib/frontend/templates/admin.dart

```diff
@@ -127,17 +127,19 @@ String renderAccountPublishersPage({
   required UserSessionData userSessionData,
   required List<PublisherSummary> publishers,
 }) {
-  final publisherListHtml = renderPublisherList(publishers, isGlobal: false);
+  final publisherListNode =
+      renderPublisherListNode(publishers: publishers, isGlobal: false);
 
   final content = renderDetailPage(
     headerHtml: _accountDetailHeader(user, userSessionData),
     tabs: [
       _myPackagesLink(),
       _myLikedPackagesLink(),
       Tab.withContent(
-          id: 'publishers',
-          title: 'My publishers',
-          contentHtml: publisherListHtml),
+        id: 'publishers',
+        title: 'My publishers',
+        contentHtml: publisherListNode.toString(),
```

Comment:
Hmm, should we not represent the content as dom nodes, and wait with the rendering till we have everything? That probably belongs in another PR.

Reply:
That's exactly the plan, and as soon as the parent template will get migrated, it will be that way.